web/src/test/java/org/apache/shiro/web/filter/mgt/PathMatchingFilterChainResolverTest.java

```diff
@@ -255,4 +255,44 @@ public void testMultipleChainsPathEndsWithSlash() {
         FilterChain resolved = resolver.getChain(request, response, chain);
         assertThat(resolved, notNullValue());
     }
+
+    /**
+     * Test asserting <a href="https://issues.apache.org/jira/browse/SHIRO-825">SHIRO-825<a/>.
+     */
+    @Test
+    public void testGetChainWhenPathEndsWithSlash() {
```

Comment:
I verified that this test passes in Shiro 1.7.0 and fails in 1.7.1 (without the fix). Thanks!